test/server/compiler-test.js

```diff
@@ -971,4 +971,27 @@ describe('Compiler', function () {
             });
         });
     });
+
+    describe('Regression', function () {
+        it('Incorrect callsite line in error report on node v0.10.41 (GH-599)', function () {
+            this.timeout(5000);
+
+            var src         = 'test/server/data/regression/gh-599/testfile.js';
+            var compiler    = new Compiler([src]);
+
+            return compiler
+                .getTests()
+                .then(function (tests) {
+                    var test = tests[0];
+
+                    return test.fn(testRunMock);
+                })
+                .then(function () {
+                    throw 'Promise rejection expected';
+                })
+                .catch(function (err) {
+                    expect(err.errMsg || err).eql('Error: Error on line - 19');
```

Comment:
Let's just keep `expect(err.errMsg)` we need to fail on fulfill, it's not important to have exact error message comparison